blockchain/tx_pool_test.go

```diff
@@ -52,18 +52,29 @@ var (
 	testTxPoolConfig TxPoolConfig
 
 	// eip1559Config is a chain config with EIP-1559 enabled at block 0.
-	eip1559Config *params.ChainConfig
+	eip1559Config params.ChainConfig
+
+	// kip71Config is a chain config with KIP-71 enabled at block 0.
+	kip71Config params.ChainConfig
 )
 
 func init() {
 	testTxPoolConfig = DefaultTxPoolConfig
 	testTxPoolConfig.Journal = ""
 
+
 	eip1559Config = params.TestChainConfig.Copy()
 	eip1559Config.IstanbulCompatibleBlock = common.Big0
 	eip1559Config.LondonCompatibleBlock = common.Big0
 	eip1559Config.EthTxTypeCompatibleBlock = common.Big0
-	fork.SetHardForkBlockNumberConfig(eip1559Config)
+	fork.SetHardForkBlockNumberConfig(&eip1559Config)
+
+	kip71Config = *params.TestChainConfig
```

Comment:
```suggestion
	kip71Config = params.TestChainConfig.Copy()
```

I think this would be better?

Reply:
It is refactored. Thanks!